client/rest/src/db/CatapultDb.js

```diff
@@ -132,6 +134,15 @@ class CatapultDb {
 		});
 	}
 
+	isConnected() {
+		return this.client.db().admin().ping()
+			.then(() => winston.verbose('Connection to mongodb is established.'))
+			.catch(err => {
+				winston.error(`failed to verify connection to mongodb: ${err}`);
+				return Promise.reject(err);
+			});
```

Comment:
not sure, but maybe this should just return promise returned by .ping(), then we could move this logic outside, down into nodeRoutes.

Reply:
Ok, let me take a look at this.